### PR TITLE
Feature/robot  debug issues

### PR DESCRIPTION
**Pull request by skristem**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot [hunk 1]
```diff
@@ -26,12 +25,12 @@ Create and edit level to verify fields
...
-    [tags]                               W-038641                 feature:Levels
+    [tags]                               W-038641                 feature:Levels    unstable
 
-    Go To Page                                       Listing                            Level__c
-    Click Special Object Button                      New
+    Go To Page                                          Listing                            Level__c
+    Click Special Object Button                         New
     Wait Until Loading Is Complete
-    Current Page Should Be                           Custom                             Level__c
+    Current Page Should Be                              Home                               Level__c
```
Comment (rjanjanam) [inline robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot]: Why the change from Custom to Home Page ? The new Page is a new create / edit levels page

Comment (skristem) [reply robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot]: I was experimenting to understand the reason for the attribute error

Comment (rjanjanam) [reply robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot]: This page object change is not needed as the page criteria do not match with the home page specification as defined by cumulus CI. Please fix the error in a different PR. As this PR's aim is to fix the failing Iframe issues.
Location should have contained '/lightning/o/Level__c/home' but it was 'https://computing-flow-5489-dev-ed.lightning.force.com/lightnin..

Comment (skristem) [reply robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot]: yes will create a new PR as soon as I merge and revert and will hand it over to you as you were working on this.
